index.html

```diff
@@ -139,16 +139,16 @@ <h2 class="title">Tonic</h2>
     </div>
   </section>
   <section class="about">
-    <section class="abtmyslf art1">
+    <h1 class="abtmyslf art1">
```

Comment:
- [ ] [OPTIONAL] Kindly make this an `h2`, `h3`, `hx`, etc but not an `h1` because we already have one in the same page.